src/microdot/microdot.py

```diff
@@ -1332,6 +1345,15 @@ async def handle_request(self, reader, writer):
                 method=req.method, path=req.path,
                 status_code=res.status_code))
 
+        # invoke the after completed request handlers
+        for handler in self.get_request_handlers(
+                req, 'after_completed_request', True):
+            res = await invoke_handler(
+                handler) or res
```

Comment:
What would be the point of `res` here? You aren't passing any arguments to this handler, and you aren't doing anything with the return value, so why have it?

Reply:
force-pushed addressing this